multisig_1_of_n_account/README.md

```diff
@@ -0,0 +1,198 @@
+        RPC-->>User/Operator: 11a. Final Transaction Result (Success)
+    else Auth Invalid
+        MultiSigContract-->>RPC: 9b. Auth check fails
+        
+        RPC-->>User/Operator: 10b. Transaction Failed (Auth Error)
+    end
+    
+    deactivate MultiSigContract
+    deactivate RPC
+```
+
+## Usage
+
+The example below sets up an asset with the contract account as the admin. The
+admin authorizes with ed25519 keys.
+
+The ed25519 keys used in the example below are:
+
+- Secret Key:
+  `0000000000000000000000000000000000000000000000000000000000000000`\
+  Public Key: `3b6a27bcceb6a42d62a3a8d02a6f0d73653215771de243a63ac048a18b59da29`
+- Secret Key:
+  `0000000000000000000000000000000000000000000000000000000000000001`\
+  Public Key: `4cb5abf6ad79fbf5abbccafcc269d85cd2651ed4b885b5869f241aedf0a5ba29`
+
+These keys are publicly viewable and not random. Do not use these keys for any
+purpose. Select your own keys and update them in the commands below when
+executing.
+
+### Install the `stellar sign-auth-ed25519` plugin
+
+Install one of the stellar sign-auth-ed25519 plugin implementations.
+
+#### Rust
+
+```
+cd stellar-cli-sign-auth-ed25519
+cargo install --locked --path .
+```
+
+#### JavaScript (Deno)
+
+```
+cd stellar-cli-sign-auth-ed25519-js
+deno install \
+    --global \
+    --name stellar-sign-auth-ed25519 \
+    --config deno.json \
+    --allow-read --no-prompt \
+    --force \
+    main.ts
+```
+
+Note: By default Deno scripts when installed have no permissions and cannot read
+or files, read environment variables, access the network, cannot execute
+commands. The `--allow-read` flag is specified to give the script permission to
+read files so that it can read .wasm dependencies in the
+`@stellar/stellar-xdr-json` package which is a Rust-built-to-wasm npm package.
+
+### Deploy the contract account
+
+```
+cd contract
+stellar contract build --out-dir out/
+stellar contract deploy \
+    --alias admin \
+    --wasm out/soroban_multisig_1_of_n_account_contract.wasm \
+    -- \
+    --signers '[
+      "3b6a27bcceb6a42d62a3a8d02a6f0d73653215771de243a63ac048a18b59da29",
+      "4cb5abf6ad79fbf5abbccafcc269d85cd2651ed4b885b5869f241aedf0a5ba29"
+    ]'
+```
+
+### Deploy asset
+
+```
+stellar keys generate issuer --fund
+stellar contract asset deploy \
+    --alias asset \
+    --asset ABC:issuer
+```
+
+### List of contracts
+
+```
+stellar contract alias ls
+```
+
+### Set admin
+
+```
+stellar contract invoke --id asset -- \
+    set_admin \
+    --new_admin admin
+```
+
+### Mint
+
+Call mint, sending to an address. The invocation will be signed by the admin
+using one of the ed25519 keys set in the constructor. Choose which key signs by
+setting the `--secret-key` option to one of the two keys above.
+
+```
+stellar contract invoke --id asset --build-only -- \
+    mint \
+    --to CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABSC4 \
+    --amount 123 \
+  | stellar tx simulate \
+  | stellar sign-auth-ed25519 \
```

Comment:
How do you "pick" which plugin to use? I want to toggle between testing the Rust plugin and JS plugin.

Reply:
By the way @fnando is currently working on improving the plugin subcommands in the cli and the next minor release will have that. Docs will probably come soon too.